odl/tomo/backends/astra_cuda.py

```diff
@@ -123,6 +123,11 @@ def astra_cuda_forward_projector_call(vol_data, geometry, proj_space,
         else:
             out.assign(tmp)
 
+    # Fix scaling issue
+    if isinstance(geometry, Parallel2dGeometry):
+        # cuda parallel2d scales linearly with linear pixel stride
```

Comment:
Good question. I realized that scaling behaviour at the very end and forgot to check that.